standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java

```diff
@@ -62,27 +63,29 @@ ClientConfig prepareValidClientConfig() {
                 .build();
     }
 
-    @Override
-    @Test
-    public void testWriteAndReadEventWithValidClientConfig() throws ExecutionException, InterruptedException {
```

Comment:
FYI: This was redundant; JUnit will execute this test since it inherits it from the super class.